src/sinks/util/auto_concurrency/tests.rs

```diff
@@ -468,10 +471,12 @@ async fn all_tests() {
         })
         .collect::<Vec<_>>();
 
-    // Then run all the tests at once
-    let tests = entries
-        .into_iter()
-        .map(|(file_path, input)| run_compare(file_path, input))
-        .collect::<Vec<_>>();
-    futures::future::join_all(tests).await;
+    entries.sort_unstable_by_key(|entry| entry.0.to_string_lossy().to_string());
+
+    time::pause();
```

Comment:
I do not see `time::resume` anywhere, `advance` work in this case?

Reply:
Yes, when the tokio timer is paused, the reference time stops. When `advance` is called, it steps forward by the duration specified and runs any timed events that were queued. `resume` ties that reference clock back to the system clock, resuming normal time flow. In this case, I want the timer to stay paused for the stability of the results, so there is no need to resume the clock. I don't think `resume` would be needed for most tests, but it would depend on exactly what is happening there.